fastmri_recon/models/utils/fourier.py

```diff
@@ -170,7 +174,9 @@ def crop_for_nufft(self, image):
 
     def op(self, inputs):
         if self.multicoil:
-            raise NotImplementedError('Multicoil NFFT is not implemented yet.')
+            image, ktraj, smaps = inputs
+            image = tf.expand_dims(image, axis=1)
+            image = image[..., 0] * smaps
```

Comment:
rather `image = image[:, None, ..., 0] * smaps`. This can even be refactored with what's underneath

Reply:
image[:, None, ..., 0] * smaps -> This adds and extra dimension, due to which the kspace nufft would not work great, we may have to remove dimesion later then.

Reply:
In my solution you wouldn't have the `expand_dims` though so I think it would be fine.